### DISCUSSION TITLE
Invalid Field or Property name

### DISCUSSION THREAD

**Discussion by mortensp** [shibayan/CsvHelper.FastDynamic#8] [Q&A] [answered]
I'm trying to read a CSV file from the web where one of the headers contains a space like: "abc efg". I call csvReader.GetDynamicRecords() and the reading proccedes just fine, but I can't access the field named "abc efg" i C# as
it's not an identifier ie. myobject."abc efg" is invalid.

**Comment by shibayan** [comment, marked as answer]
`GetDynamicRecords` returns a dynamic type, but the class implements `IDictionary<string, object>`, so it is an indexer if you cast it.

**Reply by mortensp** [reply]
Thanks, that works fine:-)
may i suggest that you implement an indexer for this, so one could write something like:

   object value = myrow["abc ef"];

**Reply by shibayan** [reply]
Thanks for the good feedback. I looked into it and it seems to be easy to implement, so I will implement it along with bug fixes in the next version.
